File: Console/Python/money.py
def main():
	p = input('Combien coûte le bien ?\n')
	m = input('Combien est-ce que tu payes ?\n')

	if(p == m):
		print('J\'ai payé ce qu\'il faut.')
	elif(p < m):
		print(f'Le caissier me rend {float(m) - float(p):0.2f} euros.')
	else:
		print(f'Je dois encore {float(p) - float(m):0.2f} euros au caissier.')

main()

input("Appuyer sur une touche pour quitter...")

File: Console/C/money.c
#include <stdio.h>

float p, m;

int main(void)
{
	printf("Combien coûte le bien ? \n"); // object's price, ex : 1€
	scanf("%f", &p);
	printf("Combien est-ce que tu payes ? \n"); // money gived, ex : 2€
	scanf("%f", &m);
	if(p == m)
	{
		printf("J'ai payé ce qu'il faut.\n");
	}
	else if(p < m)
	{
		printf("Le caissier me rend %.2f euros.\n", m - p); // money returned, ex : 1€
	}
	else
	{
		printf("Je dois encore %.2f euros au caissier.\n", p - m);
	}
	system("PAUSE");
}
